src/data/utils.ts

```diff
@@ -54,6 +53,31 @@ export function convertToCategories(d2Categories: D2Category[]): Category[] {
     }));
 }
 
+export function convertAttributeValueToDate(
```

Comment:
this is a very specific non-generic function, utils.ts probably is not be the best place.